deployments/plasma.diamond.staging.json

```diff
@@ -0,0 +1,105 @@
+{
+  "LiFiDiamond": {
+    "Facets": {
+      "0xC68B5681C55419A73b58f82d9787151decA8b1FF": {
+        "Name": "DiamondCutFacet",
+        "Version": "1.0.0"
+      },
+      "0xC7EEDC4325a7ae66ce0a0f2faa4D31B797e1D25a": {
+        "Name": "DiamondLoupeFacet",
+        "Version": "1.0.0"
+      },
+      "0x629AE13c2cEdB2E5C2f5ec482C20eA492A5CeF86": {
+        "Name": "AccessManagerFacet",
+        "Version": "1.0.0"
+      },
+      "0x781b483808dFD8cd8Cc94496Ba7035556Fb85B5B": {
+        "Name": "CalldataVerificationFacet",
+        "Version": "1.3.1"
+      },
+      "0x29fbeA2Bb8D19E2646667C1bc7877dcd71eD0c14": {
+        "Name": "DexManagerFacet",
+        "Version": "1.0.2"
+      },
+      "0xcF52515b2b4d652D2ed15aFf5e9279a03B30D9A1": {
+        "Name": "EmergencyPauseFacet",
+        "Version": "1.0.1"
+      },
+      "0x09F94360A63769995f7D66560549d9e24d47e000": {
+        "Name": "GasZipFacet",
+        "Version": "2.0.5"
+      },
+      "0xE25D3f70A3E091E78eFdc849A2C2e767f3E95378": {
+        "Name": "GenericSwapFacet",
+        "Version": "1.0.0"
+      },
+      "0x7B0309BC2011E6BC5451D18b28D19228cd45e72A": {
+        "Name": "GenericSwapFacetV3",
+        "Version": "1.0.2"
+      },
+      "0xd4F485816aecE70d3AAAD611454741C6b420f76F": {
+        "Name": "OwnershipFacet",
+        "Version": "1.0.0"
+      },
+      "0x0F9bE9173F7294d83e02DE3D6Acce5Ff73626c83": {
+        "Name": "PeripheryRegistryFacet",
+        "Version": "1.0.0"
+      },
+      "0x951f025D1c0C79DB559803c930CC9b1FD2f6DB37": {
+        "Name": "WithdrawFacet",
+        "Version": "1.0.0"
+      },
+      "0xDBf910Dac968043B78AeC8dE0cbAC3A77Bf2acB8": {
```

Comment:
not super important but could consider to remove the unused/outdated UnitFacets here